pkg/report/output/html/security.tmpl

```diff
@@ -23,7 +23,7 @@
           </div>
 
           <p class="filename">Filename: {{.Filename}}:{{.LineNumber}}</p>
-          <div class="term-container">{{. | displayExtract}}</div>
+          <pre class="term-container">{{. | displayExtract}}</pre>
```

Comment:
Dont think this needs to be changed?

Reply:
It's not required but I noticed it during my test that it respects better the spaces and it looks better.
I'm joining the before/after for comparison
WDYT?

**with Pre**
<img width="1041" alt="image" src="https://github.com/Bearer/bearer/assets/110196/a47dcb85-961c-4f82-8718-90301ecbaa25">

**with Div**
<img width="1046" alt="image" src="https://github.com/Bearer/bearer/assets/110196/90b994c6-b3e7-438f-9e93-c64b852244e5">